locale/fi/LC_MESSAGES/django.po

```diff
@@ -791,6 +874,3 @@ msgstr "Ajoneuvolta on poistettu oikeus alennukseen."
 
 msgid "Discount right ends"
 msgstr "Alennusoikeus päättyy"
-
-msgid "Announcement not found"
```

Comment:
Why is this removed?

Reply:
It was not part of translations anymore. Now added to translations.